packages/react-components/react-shared-contexts/src/index.ts

```diff
@@ -33,3 +33,6 @@ export { BackgroundAppearanceProvider, useBackgroundAppearance } from './Backgro
 export type { BackgroundAppearanceContextValue } from './BackgroundAppearanceContext';
 
 export { PortalMountNodeProvider, usePortalMountNode } from './PortalMountNodeContext';
+
+export { AnnounceProvider as AnnounceProvider_unstable, useAnnounce as useAnnounce_unstable } from './AnnounceContext';
```

Comment:
suffixing with `_unstable` doesn't actually mean you can break the API later. rather you get to create a new API that has a name without `_unstable`.

Do you think you'll need a new version of this API in the future that will drastically change?

Reply:
keeping the `_unstable` hook from discussion today:

> It might be best to keep the _unstable suffix for the component hooks for now (useFoo_unstable, useFooStyles_unstable, etc.). That way, if the signature needs to change when we "stabilize" those APIs, then there won't be an odd component that needs to have its hook be called e.g. useFoo2 because it didn't use the unstable suffix originally.